MobileClient/Client/Pages/OrderDetails.razor

```diff
@@ -0,0 +1,50 @@
+﻿@page "/Orders/{Id:long}"
+
+@inherits BasePage
+
+<AuthorizeView>
+    <NotAuthorized>
+        <RedirectToLogin/>
+    </NotAuthorized>
+    <Authorized>
+        <ConditionComponent Condition="!IsLoading">
+            <Match>
+                <ConditionComponent Condition="order != null">
+                    <Match>
+                        <div class="mainContent">
+                            <MudText Align="Align.Center">Order from @order.Date</MudText>
+                            <MudDivider Class="mb-3 mt-2" Style="border-color:#808080; border-width:0.2rem;" />
+                            <MudText Align="Align.Center" Typo="Typo.h5">Thank you for your order, your games are activated</MudText>
+                            <MudDivider Class="mb-3 mt-2" Style="border-color:#808080; border-width:0.2rem;" />
+                            <MudText Typo="Typo.h6" Class="mt-5">Keys</MudText>
+                            <div class="keysContainer">
+                                @foreach (var item in order.Items)
+                                {
+                                    <div class="keysItem">
+                                        <div class="mw-50">@item.Name</div>
+                                        <div style="text-align:right; width:40%">@(Guid.NewGuid())</div>
```

Comment:
![image](https://github.com/schlechtOptimiziert/PFAX422A-AppProjekt/assets/107460536/d26ad58c-13bd-478e-8d34-9139efa3a912)
Sieht komisch nach rechts gerückt aus.

Reply:
Der Rest sieht aber sehr gut aus! Vor allem die Einsicht in der Profiles Page gefällt mir.